src/setExchangeStatus.ts

```diff
@@ -0,0 +1,37 @@
+import { db, ExchangeStatuses, ExchangeStatus, Schema, Exchange } from './db';
+
+export async function setExchangeStatus(index: number, status: string) {
+  if (!ExchangeStatuses.includes(status as any)) {
+    console.warn("Invalid status! Available statuses are:", ExchangeStatuses);
+    return;
```

Comment:
This will be treated as suscessful call by `app.ts` (`const error = await setExchangeStatus(exchangeIndex, exchangeStatus);`)